### ISSUE TITLE
Add hot start to app home screen perf test & report how easy it is to write custom Nimbledroid tests

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/fenix#6508]
Blocked by?:
- cold/warm/hot definitions: https://github.com/mozilla-mobile/fenix/issues/5910
- Finalize Fenix cold start test: https://github.com/mozilla-mobile/fenix/issues/6507
- Finalize ND upload configuration: https://github.com/mozilla-mobile/perf-frontend-issues/issues/20

### Why/User Benefit/User Problem
A P1 metric for our release criteria is hot start to app homescreen: we should add a test for it.

While we're at it, we should report back how easy it is to write custom Nimbledroid tests to see if we should continue writing custom tests in the future, i.e. the spirit of https://github.com/mozilla-mobile/perf-frontend-issues/issues/14

### Impact
Without this, we cannot establish baseline performance or measure our proposed release criteria. We also won't know how difficult it is to write custom ND and thus if it's worth investing in.

### Acceptance Criteria (how do I know when I’m done?)
- Fenix has a hot start to app homescreen test that matches our definitions
- Link to dashboard is added to perf sync document
- ? If needed, file follow-up to add this information into our dashboards
- Issue include description of how easy it is to write custom ND tests
- Team is notified about how easy it is to write custom ND tests

**Comment by mcomella** [comment]
Findings on adding custom ND tests: the official docs say we need to upload the Espresso APK with the tests we care about. We'll need to make sure to exclude non-perf espresso tests. No-Jun mentioned this infrastructure may not work yet: if so, contact me. We'll want to upload to our official ND channel (i.e. the "app" in their interface owned by our generic Nimbledroid user). We may have to build out some infrastructure to get this working (i.e. uploading) in the official channel.

**Comment by mcomella** [comment]
(copy-paste) Not a priority: closing.
